mne_icalabel/features/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Features for the ICLabel"""
+from features import topomap
```

Comment:
```suggestion
from .topomap import topographic_map
```

Here you need a relative import. You are in the `features` modules (which corresponds to `.`) and you need to import `topographic_map` from `topomap.py`. So `.topomap` -> `topomap.py` file in the current module (`features`).
That will also fix the tests and run them.

Reply:
With the new names, it would be 

```suggestion
from .topomap import get_topomap, get_topomaps  # noqa: F401
```

I'm also adding the `noqa` flag to disable the F401 error raised by static code analysis tool. c.f. https://www.flake8rules.com/rules/F401.html